spec/fortnox/api/types/housework_types_spec.rb

```diff
@@ -42,49 +45,46 @@
         let(:error_message) { 'Skattereduktion för den valda typen av husarbete har upphört.' }
 
         it 'raises an error' do
```

Comment:
This could quite easily be rewritten to
```
it is_expected.to raise_error(Fortnox::API::RemoteServerError, error_message)
```
since you don't really need the description there ...